README.md

```diff
@@ -1,38 +1,42 @@
-a Java 1.4 codebase to Java 5, taking advantage of generics for type-safety and `java.util.concurrent` classes for improved
-performance.  Notable improvements are:
 * Greater use of low-contention lock collections throughout the code and the removal of most large-grained locks.
 * A new connection pool with a focus on zero-wait connection acquisition and concurrent expansion/contraction. For one large
-workload compared to BTM 2.1 total connection wait time went from 76 seconds to 372ms, and lock contentions went from 26269
+workload compared to BTM 2.1, total connection wait time went from 76 seconds to 372ms, and lock contentions went from 26269
 to just 21.
-* High-performance proxies around `javax.sql` entities via bytecode generation with support for Javassist, and cglib, with 
-fallback to `java.lang.reflect.proxy`.  Javassist and cglib offer substantial performance increases and are strongly recommended.
-* High-performance transaction log journaling using a new design that allows concurrent appenders through a 
-write-reservation model.  The new journal is 3-14x faster than BTM 2.1.
-* Support for all levels of JDBC upto and including JDBC 4.1
-* OSGi support
+* High-performance proxies around `javax.sql` entities via bytecode generation with support for Javassist and cglib, with 
+fallback to `java.lang.reflect.proxy`. Javassist and cglib offer substantial performance increases and are strongly recommended.
+* High-performance transaction log journaling using a new design that allows concurrent appenders through a write-reservation model. The new journal is 3 to 14 times faster than BTM 2.1.
+* Support for all levels of JDBC up to and including JDBC 4.1.
+* OSGi support.
 * A change of license from LGPL v3 to Apache 2.
 
-#### General Information ####
-* [Overview](https://github.com/bitronix/btm/wiki/Overview)
-* [FAQ](https://github.com/bitronix/btm/wiki/FAQ)
+## General information
+
+For general information, see the following:
+
+* [Overview](https://github.com/scalar-labs/btm/wiki/Overview)
+* [FAQ](https://github.com/scalar-labs/btm/wiki/FAQ)
+
+## Configuration
+
+For details about configuring BTM, see the following:
+
+* [Transaction manager configuration](https://github.com/scalar-labs/btm/wiki/Transaction-manager-configuration)
+* [Resource loader configuration](https://github.com/scalar-labs/btm/wiki/Resource-loader-configuration)
+
+## Credits
 
-#### Configuration ####
-* [Transaction manager configuration](https://github.com/bitronix/btm/wiki/Transaction-manager-configuration)
-* [Resource loader configuration](https://github.com/bitronix/btm/wiki/Resource-loader-configuration)
+Scalar, Inc., would like to thank [@bitronix](https://twitter.com/bitronix) and [@BrettWooldridge](https://twitter.com/BrettWooldridge) for their innovative work on BTM. They have gracefully transferred ownership of this repository to us so that we can continue to maintain and build on this project.
```

Comment:
How about something like this? Please correct my English.
```suggestion
BTM had been created and maintained by the Bitronix team and its community, especially [@bitronix](https://twitter.com/bitronix) and [@BrettWooldridge](https://twitter.com/BrettWooldridge).
Scalar, Inc. took over the great project so that we can continue to maintain and build on the project.
We would like to thank [@bitronix](https://twitter.com/bitronix) and [@BrettWooldridge](https://twitter.com/BrettWooldridge) for their innovative work on BTM.
```

Reply:
Sounds good! I've made some minor edits to your suggestion and moved this section near the top of this README.